meinberlin/apps/contrib/templates/meinberlin_contrib/includes/form_field.html

```diff
@@ -1,26 +1,33 @@
 {% load i18n widget_tweaks %}
 
 <div class="form-group {% if add_class %}{{ add_class }}{% endif %}">
-    <label for="{{ field.id_for_label }}" {% if tabindex %} tabindex="{{ tabindex }}"{% endif %}>
-        {{ field.label }}{% if field.field.required %}<span role="presentation" title="{% translate 'This field is required' %}">*</span>{% endif %}
-        {% block after_label %}
-        {% endblock %}
+    <label for="{{ field.id_for_label }}" {% if tabindex %} tabindex="{{ tabindex }}" {% endif %} class="form-label">
+        {{ field.label }}
+        {% if field.field.required %}
+            <span class="icon-required" aria-hidden="true">*</span>
+            <span class="aural">{% translate 'This field is required' %}</span>
+        {% endif %}
+        {% block after_label %}{% endblock after_label %}
     </label>
-    {% if field.help_text %}
-    <div class="form-hint">
-        {{ field.help_text }}
-    </div>
-    {% endif %}
+
     {% block field %}
         <div class="widget widget--{{ field|widget_type }}">
             {{ field|add_error_attr:"aria-invalid:true" }}
         </div>
-    {% endblock %}
+    {% endblock field %}
+    
+
+    {% if field.help_text %}
```

Comment:
This will move the helptexts for all fields in the dashboard and break the styling, so either we also copy this file or at least add the `form-hint` class back in so the styling is still there.

Reply:
sorry, could have added a link: check http://127.0.0.1:8003/dashboard/organisations/liqd/plans/create/

Reply:
yeah, it was nice while we had the same styles everywhere but now with the redesign they are more of a burden. I think having one for user_facing and one for dashboard takes away the complexity for now and is easy to do